exports.js

```diff
@@ -1038,7 +1038,11 @@ module.exports = {
         'amsDiagnosticLogsEnabled'      : require(__dirname + '/plugins/azure/mediaServices/amsDiagnosticLogsEnabled.js'),
         'amsPublicAccessDisabled'       : require(__dirname + '/plugins/azure/mediaServices/amsPublicAccessDisabled.js'),
         'amsManagedIdentityEnabled'     : require(__dirname + '/plugins/azure/mediaServices/amsManagedIdentityEnabled.js'),
-        'amsClassicApiDisabled'         : require(__dirname + '/plugins/azure/mediaServices/amsClassicApiDisabled.js')
+        'amsClassicApiDisabled'         : require(__dirname + '/plugins/azure/mediaServices/amsClassicApiDisabled.js'),
+    
+        'appConfigurationManagedIdentity': require(__dirname + '/plugins/azure/appConfigurations/appConfigurationManagedIdentity.js'),
```

Comment:
```suggestion
        'appConfigurationManagedIdentity': require(__dirname + '/plugins/azure/appConfigurations/appConfigurationManagedIdentity.js'),
```
can we change it to appConfigManagedIdentity?